Taskfile.Shared.yml

```diff
@@ -62,56 +62,31 @@ tasks:
     label: "{{.TASK}}-{{.NAME}}"
     dir: coprocessors/{{.NAME}}
     cmds:
-      - docker run -d --rm --name {{.NAME}}-co --env PORT="{{.COPROCESSOR_PORT}}" -m 1g --cpus=1 -p {{.COPROCESSOR_PORT}}:{{.COPROCESSOR_PORT}} {{.NAME}}-co
+      - docker run -d --rm --name coprocessor --net=task --env PORT="8000" -p 8000:8000 -m 1g --cpus=1 {{.NAME}}-co
```

Comment:
Any reason this was changed?

Reply:
Because that name is used in Docker networking, so the router config uses `http://coprocessor:8000` now (instead of localhost). It's possible we can switch back to localhost if we can configure GHA docker to use the `docker.network.internal` stuff but this was faster. (see router_co.yaml files)